packages/cli/src/cli/services/upgrade.ts

```diff
@@ -48,7 +48,12 @@ export async function upgrade(
 }
 
 async function getProjectDir(directory: string): Promise<string | undefined> {
-  const configFile = await findPathUp(['shopify.app.toml', 'hydrogen.config.js', 'hydrogen.config.ts'], {
+  const configFiles = ['shopify.app{,.*}.toml', 'hydrogen.config.js', 'hydrogen.config.ts']
```

Comment:
what does the `,` on `{,.*}` does?

Reply:
The comma , is necessary to separate the different options within the brace expansion {], in this case the empty string and any string that starts with '.'

Reply:
Wouldn't it work without it? `.*` should match empty strings too